src/app/shared/theme-picker/theme-storage/theme-storage.ts

```diff
@@ -16,15 +16,24 @@ export class ThemeStorage {
   public onThemeUpdate: EventEmitter<DocsSiteTheme> = new EventEmitter<DocsSiteTheme>();
 
   public storeTheme(theme: DocsSiteTheme) {
-    window.localStorage[ThemeStorage.storageKey] = JSON.stringify(theme);
+    try {
+      window.localStorage[ThemeStorage.storageKey] = JSON.stringify(theme);
```

Comment:
I don't think `sessionStorage` would be available in this case either (at least it doesn't look like it is on IE11).